tigerpath/majors_and_certificates/majors/NES_2018.json

```
@@ -0,0 +1,213 @@
+                    "NES 343",
+                    "NES 345",
+                    "NES 350",
+                    "NES 358",
+                    "NES 365",
+                    "NES 416",
+                    "NES 419",
+                    "NES 433",
+                    "NES 437",
+                    "NES 438"
+                  ]
+                },
+                {
+                  "name": "Literature",
+                  "max_counted": 1,
+                  "min_needed": 1,
+                  "explanation": "<i>Editor's Note: This does not represent a comprehensive list of courses that fulfill this requirement as courses change from semester to semester.</i>",
+                  "course_list": [
+                    "NES 202",
+                    "NES 214",
+                    "ARA 401",
+                    "ARA 404",
+                    "HEB 301",
+                    "HEB 302",
+                    "PER 301"
+                  ]
+                },
+                {
+                  "name": "Social Sciences",
+                  "max_counted": 1,
+                  "min_needed": 1,
+                  "explanation": "<i>Editor's Note: This does not represent a comprehensive list of courses that fulfill this requirement as courses change from semester to semester.</i>",
+                  "course_list": [
+                    "NES 265",
+                    "NES 315",
+                    "NES 345",
+                    "NES 347",
+                    "NES 348",
+                    "NES 363"
+                  ]
+                },
+                {
+                  "name": "Religion",
+                  "max_counted": 1,
+                  "min_needed": 1,
+                  "explanation": "<i>Editor's Note: This does not represent a comprehensive list of courses that fulfill this requirement as courses change from semester to semester.</i>",
+                  "course_list": [
+                    "NES 220",
+                    "NES 236",
+                    "NES 240",
+                    "NES 268",
+                    "NES 269",
+                    "NES 334",
+                    "NES 336",
+                    "NES 339",
+                    "NES 340",
+                    "NES 348",
+                    "NES 356",
+                    "NES 435"
+                  ]
+                }
+              ]
+            }
+          ]
+        },
+        {
+          "name": "8 Departmentals",
+          "max_counted": 1,
+          "min_needed": 8,
+          "explanation": "NES majors take eight departmental courses in Near Eastern Studies.",
+          "double_counting_allowed": true,
+          "req_list": [
+            {
+              "name": "NES-Prefix Courses",
+              "max_counted": null,
+              "min_needed": 5,
+              "explanation": "At least five departmentals must be NES-prefix or language courses.",
+              "course_list": [
+                "NES *"

```

Again, it says language courses count. I think that means languages in the department, but I might be wrong.
```suggestion
                "NES *",
                "ARA *",
                "HEB *",
                "PER *",
                "TUR *"
```